A32-xds-circuit-breaking.md

```diff
@@ -116,9 +130,11 @@ TODO
 
 ## Other parameters considered
 
-There is a group of other parameters in the xDS circuit breakers, but most of
-them do not apply to gRPC's use case, and they will be ignored in the 
-implementation. The following parameters have been considered:
+There is a group of other parameters in the xDS circuit breakers, but they
```

Comment:
I would phrase this a little differently.  I think the two categories are:

1. The parameters are fundamentally not applicable to gRPC in its current form.  It's possible that gRPC could evolve to make them relevant, in which case we would reevaluate them, but it's also possible that these will simply never be supported.
2. The parameters are applicable to gRPC, but the implementation is not trivial, and we don't want to invest the resources to implement them until/unless we know that they will be useful enough to make that investment worthwhile.  We will reevaluate these as we get feedback from users.

I suggest splitting this list of parameters into two lists, one for each of the categories above.

I think the following parameters are in category 1:
- `max_connections` and `max_requests_per_connection`: gRPC only ever creates a single connection to a given endpoint.  But in the future, we might provide a new LB policy to allow multiple connections to address MAX_CONCURRENT_STREAMS limits, at which point it might make sense to support this.
- `max_connection_pools`: gRPC does not currently have a concept of connection pools the way that Envoy does.  This will likely never apply to gRPC.

I think the following parameters are in category 2:
- `max_pending_requests`: Your existing description of this one is good.
- `connect_timeout`: In addition to what you've already written for this one, it also has particular challenges in C-core due to the way we share subchannels between channels.
- `max_retries` and `max_budget`: These will be addressed when we eventually add support for configuring retries via xDS.

Reply:
Updated to separate them into two categories.